src/js/views/patients/actions/actions_views.js

```diff
@@ -141,7 +140,7 @@ const OwnerComponent = Droplist.extend({
     }
     return {
       modelEvents: {
-        'change:_role change:_clinician': 'render',
+        'change:_role change:_clinician change:_owner': 'render',
```

Comment:
Just a note, we need to retain `change:_role change:_clinician` until the flows models are also migrated to use owner.